CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
```

Comment:
I think it should be `Changed` :sweat_smile: 

I will change tomorrow.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/6465/commits/72ee22f570795fa2ab6e4c56957f30f55f882e9a